Vagrantfile

```diff
@@ -59,5 +59,7 @@ Vagrant.configure("2") do |box|
     # forward leap_web ports
     config.vm.network "forwarded_port", guest: 80,  host:8080
     config.vm.network "forwarded_port", guest: 443, host:4443
+    # forward pixelated ports
```

Comment:
Didn't you noticed during testing that port 8080 (that should point to port 80 on the vm) doesn't work ? If so, we should take it out and can then use port 8080 for pixelated.